### DISCUSSION TITLE
Size issues

### DISCUSSION THREAD

**Discussion by FelixZ-L** [OleehyO/TexTeller#85] [Q&A]
After changing the data set, I got [rank0]: RuntimeError: Input and output sizes should be greater than 0, but got input (H: 1, W: 592) output (H: 0, W: 448) What is the reason?

**Comment by OleehyO** [comment]
TexTeller scales images proportionally to 448 during training. Therefore, if there are some unusual data points (most likely due to data not being properly cleaned), such as the example you provided with an image having dimensions of H:1 and W:592, the height becomes 0 after scaling, leading to an error.

**Reply by FelixZ-L** [reply]
OK, thanks for taking the time to answer.
